scripts-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mangoweb/scripts-base",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Tohle je čistší dělat separé commitem.

Reply:
Uhh, asi máš pravdu. Mám to ještě změnit a force-pushnout bez toho?